ydb/core/blobstorage/dsproxy/dsproxy_request_reporting.cpp

```diff
@@ -0,0 +1,46 @@
+#include "dsproxy_request_reporting.h"
+
+namespace NKikimr {
+
+static std::array<std::atomic<bool>, 7> ReportPermissions;
```

Comment:
If a new EGetHandleClass is added there will be an out-of-bounds access to the array in AllowToReport

Reply:
it would be more convenient if you used two separate arrays here and calculated their sizes using NKikimrBlobStorage::EPutHandleClass_MAX and NKikimrBlobStorage::EGetHandleClass_MAX macros